corehq/apps/hqwebapp/management/commands/migrate_app_to_bootstrap5.py

```diff
@@ -142,6 +142,12 @@ def handle(self, app_name, **options):
             )
             return
 
+        if self.skip_all and (js_name or template_name):
```

Comment:
Why is this?

Reply:
because we want the user to confirm the correct file paths were picked up. often this is only one file anyway, so `--skip-all` doesn't make much of a difference. Additionally, we want to prompt the user differently if the file is a `bootstrap3` re-migration.